sway/sway-ipc.7.scd

```diff
@@ -379,6 +379,14 @@ node and will have the following properties:
 |- shell
 :  string
 :  (Only views) The shell of the view, such as _xdg\_shell_ or _xwayland_
+|- inhibit_idle
+:  boolean
+:  (Only views) Whether the view is inhibiting the idle state
+|- idle_inhibitors
+:  object
+:  (Only views) An object containing the state of the _application_ and _user_ idle inhibitors.
+    _application_ can be _enabled_, _disabled_ (not supported yet) or _none_.
```

Comment:
Can you just drop the disabled bit instead of documenting something that isn't currently supported? If/when support for disabling an application idle inhibitor gets implemented, this can be added in.

The second example below will also need to be updated.